packages/guidelines/README.md

```diff
@@ -0,0 +1,333 @@
+# GUIDELINES
```

Comment:
why having this guidelines here? 🤔

Reply:
which task is this?

Reply:
just add this here becauseI thought it matches with this repo purposal!

Reply:
makes sense! we can update them after